src/app/entry/delete/dialog/delete-entry-dialog.component.html

```diff
@@ -7,9 +7,9 @@ <h1 mat-dialog-title>Are you sure you want to delete this Dockstore {{ term }}?<
   </p>
   <p>A {{ term }} can be deleted if it:</p>
   <ul>
-    <li>Has never been published,</li>
```

Comment:
I mean, if we're fixing punctuation, these don't need periods or commas
See the "Research office" example of https://www.imperial.ac.uk/brand-style-guide/writing/grammar/punctuation/bullet-points/

Reply:
The bullet items "complete the introductory stem", thus each forming full sentences, so according to this, at least, they should be punctuated:
https://www.microsoft.com/en-us/microsoft-365-life-hacks/writing/punctuating-bullet-point-lists

Reply:
If you're going with that guide, the first word of each shouldn't be capitalized then